selfdrive/frogpilot/frogpilot_functions.py

```diff
@@ -191,33 +190,6 @@ def setup_frogpilot(build_metadata):
     run_cmd(["sudo", "cp", frogpilot_boot_logo, boot_logo_location], "Successfully replaced boot logo", "Failed to replace boot logo")
     run_cmd(["sudo", "mount", "-o", f"remount,{stock_mount_options}", "/"], "Successfully restored stock mount options", "Failed to restore stock mount options")
 
-  persist_comma_path = Path("/persist/comma")
-  backup_comma_path = Path("/data/backup_comma")
-  if persist_comma_path.exists():
-    shutil.copytree(persist_comma_path, backup_comma_path, dirs_exist_ok=True)
-    print("Successfully backed up /persist/comma to /data/backup_comma")
-
-  persist_params_path = Path("/persist/params")
-  if persist_params_path.exists() and persist_params_path.is_dir():
-    shutil.rmtree(persist_params_path)
-    print("Successfully deleted /persist/params")
-
-  persist_tracking_path = Path("/persist/tracking")
-  if persist_tracking_path.exists() and persist_tracking_path.is_dir():
```

Comment:
I'm not sure how long this migration code has existed for so far, so maybe you don't want to remove it yet